### ISSUE TITLE
Octodash setup error "Http failure response for http://localhost:5000/api/connection: 403 FORBIDDEN"

### ISSUE THREAD

**Issue by Labdoc1957** [UnchartedBull/OctoDash#1090]
**What doesn't work?**

**What did you already try?**
A clear and concise description of what you tried to make OctoDash work.

**General Information:**
 - Hardware [e.g. Raspberry Pi, if you have layout issues please also include your screen resolution]
 - OS Info [e.g. Raspbian Buster Lite, please also indicate if you used OctoPi]
 - OctoDash Version [e.g. v1.0.0]
 - OctoPrint Version [e.g. v1.0.0]

**Additional context**

**Comment by Labdoc1957** [comment]
Octodash setup error "Http failure response for http://localhost:5000/api/connection: 403 FORBIDDEN"

So I set up Octoprint on my Pi4B 4Gb and on my Pi4B 8Gb which are working as expected.  I'm trying to set up a BTT PiTFT50 on my 4Gb Pi.  I went through the install process and the OctoDash starts up automatically, however, I'm unable to get past the setup screens.  I go through all of the screens and the whole thing comes to a crashing halt on the validation (final) screen. Any thoughts on how to get past this?
![IMG_8297](https://user-images.githubusercontent.com/71477873/96373446-86aa1180-113a-11eb-99b9-566532cceb5c.jpeg)
